ibis/backends/duckdb/tests/test_client.py

```diff
@@ -132,3 +132,29 @@ def test_create_table_with_timestamp_scales(con, scale):
     schema = ibis.schema(dict(ts=dt.Timestamp(scale=scale)))
     t = con.create_table(gen_name("duckdb_timestamp_scale"), schema=schema, temp=True)
     assert t.schema() == schema
+
+
+def test_config_options(con):
+    def _get_null_value():
```

Comment:
Testing this here seems unnecessary since it's testing that a particular DuckDB sql query works independent of ibis whereas the `con.settings` calls are testing an ibis API.

Reply:
I was trying to show that the underlying setting is mutated, but I think I've got a better way to demonstrate that